lib/states/wi/madison.R

```diff
@@ -8,86 +8,136 @@ source("common.R")
 # each year, this seems reasonable.
 load_raw <- function(raw_data_dir, n_max) {
   # NOTE: "IBM" is the officers department ID
-  cit <- load_single_file(
+  cit_old <- load_single_file(
     raw_data_dir,
     "mpd_traffic_stop_request_citations.csv",
     n_max
   )
-  warn <- load_single_file(
+  warn_old <- load_single_file(
     raw_data_dir,
     "mpd_traffic_stop_request_warnings.csv",
     n_max
   )
+  
+  cit_new <- load_single_file(
+    raw_data_dir,
+    "elcis_with_demographics.csv",
+    n_max
+  )
+  
+  warn_new <- load_single_file(
+    raw_data_dir,
+    "warnings_with_demographics.csv",
+    n_max
+  )
+  
+  ## NOTE: The new data starts in Jan. 2018 
+  ## and the old data covers through Sept. 2017, 
+  ## so we're still missing Oct.-Dec. 2017
+  new_d <- bind_rows(
+    cit_new$data,
+    warn_new$data
+  )
```

Comment:
Is there a reason why you created `new_d` with a `bind_rows` here, instead of binding them all below in row 43?

Reply:
I think perhaps at the time I thought it might be more clear this way but I see your point. I'll go ahead and bind them all together below.